### DISCUSSION TITLE
Deserialize list of `T` using `System.Text.Json` with additional conditions

### DISCUSSION THREAD

**Discussion by MartyIX** [dotnet/runtime#62688] [Q&A] [answered]
Hi,

I know I can deserialize `List<T>` easily using STJ. I would love to know if I can actually set capacity of the list upfront or not. I know I can write a custom converter. I'm just curious if I can set the list capacity without custom converter.

Thanks!

**Comment by huoyaoyuan** [comment, marked as answer]
@dotnet/area-system-text-json 

I believe that if you can't find an attribute for it, then it's just not supported.
